lib/db-utils/migrations.ts

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs-extra';
+import makeDebug from 'debug';
+import Database from 'better-sqlite3';
+import {
+    DB_VERSION_TABLE_NAME,
+    DB_CURRENT_VERSION,
+    DB_SUITES_TABLE_NAME,
+    DB_COLUMNS,
+    VERSION_TABLE_COLUMNS,
+    LabeledVersionRow
+} from '../constants/database';
+import {isEqual} from 'lodash';
+import {createTableQuery} from './common';
+const debug = makeDebug('html-reporter:db-migrations');
+
+export const getDatabaseVersion = (db: Database.Database): number | null => {
+    debug('getDatabaseVersion');
+    try {
+        const versionTableExists = db.prepare(
+            `SELECT name FROM sqlite_master WHERE type='table' AND name=?`
+        ).get(DB_VERSION_TABLE_NAME);
+        debug('versionTableExists: ', versionTableExists);
```

Comment:
It seems that you don't have to put a space here and below. In this case two spaces will appear in the log